client/src/global/components/composed/EntityGroup/styles.js

```diff
@@ -57,9 +48,10 @@ export const GroupHeader = styled.div`
     `}
 `;
 
-export const HeaderText = styled.span`
+export const HeaderText = styled.h2`
   ${headingPrimary}
+  margin-inline-start: -15px;
   margin-bottom: 0;
-  color: var(--color-neutral-text-extra-dark);
+  color: var(--strong-color);
```

Comment:
Not essential, but if you define this on `GroupHeader` you only have to do it once. The icon and HeaderText will just inherit it.